src/skmultiflow/data/time_manager.py

```diff
@@ -1,156 +1,156 @@
-
-        """
-
-        # get samples that have label available
-        samples = self.queue[self.queue['available_time'] <= self.timestamp]
-        # remove these samples from queue
-        self.queue = self.queue[self.queue['available_time'] > self.timestamp]
-        # get X, y_true and y_pred
-        X = np.array(samples["X"].values.tolist())
-        y_true = np.array(samples["y_true"].values.tolist())
-        y_pred = np.array(samples["y_pred"].values.tolist())
-        # return X, y_true and y_pred for the dequeued samples
-        return X, y_true,  y_pred
-
-    def update_queue(self, X, y_true, y_pred, sample_weight, arrival_time, available_time):
-        # check if weight is None to create a list
-        if sample_weight is None:
-            # set _sample_weight as False, indicating that there is no weight
-            self._USE_SAMPLE_WEIGHT = False
-            sample_weight = np.full(X.shape[0], None)   # noqa
-        # create daraframe for current samples
-        frame = pd.DataFrame(list(zip(X, y_true, y_pred, sample_weight,
-                                      arrival_time, available_time)),
-                             columns=self._COLUMN_NAMES)
-        # append new data to queue
-        self.queue = self.queue.append(frame)
-        # sort queue
-        self._sort_queue()
-
-    def has_more_samples(self):
-        """ Checks if queue has more samples.
-
-        Returns
-        -------
-        Boolean
-            True if queue has more samples.
-
-        """
-
-        return self.queue.shape[0] > 0
-
-    def next_sample(self, batch_size=1):
-        """ Returns next sample from the queue.
-
-        If there is enough instances to supply at least batch_size samples, those
-        are returned. Otherwise, the remaining are returned.
-
-        Parameters
-        ----------
-        batch_size: int (optional, default=1)
-            The number of instances to return.
-
-        Returns
-        -------
-        tuple or tuple list
-            Returns the next batch_size instances.
-            For general purposes the return can be treated as a numpy.ndarray.
-
-        """
-        if self.queue.shape[0] - batch_size >= 0:
-            samples = self.queue[:batch_size]
-        else:
-            samples = self.queue
-        # get X
-        X = np.array(samples["X"].values.tolist())
-        # get y_true
-        y_true = np.array(samples["y_true"].values.tolist())
-        # get y_pred
-        y_pred = np.array(samples["y_pred"].values.tolist())
-        # check if sample_weight are being used
-        if self._USE_SAMPLE_WEIGHT:
-            sample_weight = np.array(samples["weight"].values.tolist())
-        else:
-            sample_weight = None
-        # remove samples from queue
-        self._cleanup_samples(batch_size)
-        # return X, y_true, y_pred, and weight for the dequeued samples
-        return X, y_true, y_pred, sample_weight
+import numpy as np
+import pandas as pd
```

Comment:
Does this entire file was added without changes?

Reply:
I accidentally changed the file encoding from `CRLF` to `LF`. I deleted only one white space. Let me revert the changes.

Reply:
It's done! Now you can see the difference correctly.

Reply:
Perfect!